dask_cuda/cuda_worker.py

```diff
@@ -144,8 +144,8 @@ def del_pid_file():
                 warnings.warn(
                     "When using NVLink we recommend setting a "
                     "`rmm_pool_size`.  Please see: "
-                    "https://dask-cuda.readthedocs.io/en/latest/ucx.html"
-                    "#important-notes for more details"
+                    "https://docs.rapids.ai/api/dask-cuda/nightly/ucx/ "
+                    "for more details"
```

Comment:
Seemed like the section the original link was pointing to doesn't exist anymore, is there any specific place on the UCX integration docs we'd like to point this to?

Reply:
This links looks like the correct one to me.